roles/default/kiali-deploy/defaults/main.yml

```diff
@@ -43,6 +43,14 @@ kiali_defaults:
       ldap_use_ssl: false
       ldap_user_filter: "(cn=%s)"
       ldap_user_id_key: "cn"
+    openid:
+      authentication_timeout: 300
+      authorization_endpoint: ""
+      client_id: ""
+      insecure_skip_verify_tls: false
+      issuer_uri: ""
+      scopes: ["profile", "email"]
```

Comment:
I know that in the server code you forcibly add "openid" if it doesn't exist, but for clarity, shouldn't you just put "openid" in here explicitly? That way people seeing this will know, "ah, ok, this does have openid scope" - otherwise, they will be asking the question, "Why isn't openid in here? Do I need to add it?" I realize they can read it in the above comments, but I think it is best to also explicitly set it here to. Again, that just makes it clear.

Reply:
Fixed. openid is added to the list.